src/mame/drivers/pgm.cpp

```diff
@@ -3951,15 +3951,15 @@ ROM_END
 ROM_START( svgpcb )
 	ROM_REGION( 0x600000, "maincpu", 0 ) /* 68000 Code */
 	ROM_LOAD16_WORD_SWAP( "svg_bios.u49",    0x000000, 0x020000, CRC(3346401f) SHA1(28bd730b6026c1e521c95072d33c7bdcd19c1460) )
-	ROM_LOAD16_WORD_SWAP( "svg_v100jp.u50",  0x100000, 0x080000, CRC(8d0405e4) SHA1(b6175c9ffeaac531d28e7845cb34c673476e286a) )
+	ROM_LOAD16_WORD_SWAP( "svg_v100jp.u50",  0x100000, 0x080000, CRC(8d0405e4) SHA1(b6175c9ffeaac531d28e7845cb34c673476e286a) ) // V100 03/14/05 20:04:08
```

Comment:
Why do some of your comment like this one have the version before the date (V100 03/14/05 20:04:08) while the one above has the version after the date (06/20/05 11:36:15 V100)?  Is there a difference in the way the games display the build information?  If not, the comments should be consistent.

Reply:
V100 03/14/05 20:04:08 is M68K code revision shown at operator setting menu, 06/20/05 11:36:15 V100 is ARM code revision shown at warning screen. ex: svgtw and svgpcb shares same 68K data, and revision is same on operator setting menu.
most of games are similar; theglad, killbldp, happy6 - these games are shown M68K and Internal ARM code version info in bottom of operator setting menu(sometimes blinking), and Build date of ARM external code is shown at warning screen.
when ARM external ROM hasn't needed/implemented or games released on kov-era to kov2-era, it's only shown M68K version at same case.